### PR TITLE
chore(helm): default image tag to AppVersion

### PR DESCRIPTION
**Pull request by oleobal**
## Description

This removes some redundancy by making image tag default to AppVersion (so it only needs setting once rather than twice)

See https://github.com/Substra/substra-backend/pull/596, which is the same but for the backend. The frontend already does that.

## How has this been tested?

```
git switch main
helm template charts/orchestrator > old-render.yaml
git switch chore-helm/default-appversion
helm template charts/orchestrator > new-render.yaml
diff old-render.yaml new-render.yaml
```

## Checklist

- [x] [changelog](../CHANGELOG.md) was updated with notable changes
- [x] documentation was updated

### CODE REVIEW

File: charts/orchestrator/templates/_helpers.tpl [hunk 1]
```diff
@@ -167,3 +167,17 @@ Return the proper image name
 {{- printf "%s:%s" .repository .tag -}}
 {{- end -}}
 {{- end -}}
```
Comment (guilhem-barthes) [inline charts/orchestrator/templates/_helpers.tpl:169]: It looks like the block from L160 to L169 is not used anymore, if it's the case, should we remove it?

Comment (oleobal) [reply charts/orchestrator/templates/_helpers.tpl:169]: It's not used anymore, but I was thinking this way the `common` stuff stays the same between orc and backend.

I'm fine with removing it though, if you think it better.

Comment (oleobal) [reply charts/orchestrator/templates/_helpers.tpl:169]: @guilhem-barthes watcha think, keep it or drop it?

**Review by guilhem-barthes — APPROVED** [review]
Thanks for your work!

**Pull request merged**